paasta_tools/cli/cmds/rerun.py

```diff
@@ -0,0 +1,139 @@
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import argparse
+import datetime
+
+from paasta_tools.cli.cmds.status import get_actual_deployments
+from paasta_tools.cli.cmds.status import get_planned_deployments
+from paasta_tools.cli.cmds.status import list_deployed_clusters
+from paasta_tools.cli.utils import execute_chronos_rerun_on_remote_master
+from paasta_tools.cli.utils import figure_out_service_name
+from paasta_tools.cli.utils import lazy_choices_completer
+from paasta_tools.cli.utils import list_instances
+from paasta_tools.cli.utils import list_services
+from paasta_tools.utils import list_clusters
+from paasta_tools.utils import PaastaColors
+from paasta_tools.utils import SPACER
+
+
+EXECUTION_DATE_FORMAT = "%Y-%m-%dT%H:%M:%S"
+
+
+def type_execution_date(value):
+    try:
+        parsed = datetime.datetime.strptime(value, EXECUTION_DATE_FORMAT)
+    except ValueError:
+        raise argparse.ArgumentTypeError('must be in the format "%s"' % EXECUTION_DATE_FORMAT)
+    return parsed
+
+
+def add_subparser(subparsers):
+    rerun_parser = subparsers.add_parser(
+        'rerun',
+        help="Re-run a scheduled PaaSTA job",
+        description=(
+            "'paasta rerun' creates a copy of the specified PaaSTA scheduled job and executes it immediately. "
+            "Parent-dependent relationships are ignored: 'pasta rerun' only executes individual jobs."
+        ),
+        epilog=(
+            "Note: This command requires SSH and sudo privileges on the remote PaaSTA "
+            "masters."
+        ),
+    )
+    rerun_parser.add_argument(
+        '-v', '--verbose',
+        action='count',
+        dest="verbose",
+        default=0,
+        help="Print out more output regarding the operation."
+    )
+    rerun_parser.add_argument(
+        '-s', '--service',
+        help='The name of the service you wish to operate on.',
+        required=True,
+    ).completer = lazy_choices_completer(list_services)
+    rerun_parser.add_argument(
+        '-i', '--instance',
+        help='Name of the scheduled job (instance) that you want to rerun.',
+        required=True,
+    ).completer = lazy_choices_completer(list_instances)
+    rerun_parser.add_argument(
+        '-c', '--clusters',
+        help="A comma-separated list of clusters to rerun the job on. Defaults to rerun on all clusters.\n"
+             "For example: --clusters norcal-prod,nova-prod"
+    ).completer = lazy_choices_completer(list_clusters)
+    rerun_parser.add_argument(
+        '-d', '--execution_date',
+        help="The date the job should be rerun for. Expected in the format %%Y-%%m-%%dT%%H:%%M:%%S .",
+        required=True,
```

Comment:
What do you think about defaulting this to "now"? So if a job fails and you want to rerun it right away, you can easily?